From f63b15e690581d898da56ef2540b6199f5728e58 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Tue, 11 Feb 2020 19:14:53 +0100
Subject: [PATCH] tests/Makefile.am: don't add -lpthread

Don't add -lpthread unconditionally to oomtest_LDADD to avoid a build
failure without threads, it will be added to LIBS by configure.ac if
needed

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Retrieved from:
https://github.com/cminyard/gensio/commit/f63b15e690581d898da56ef2540b6199f5728e58]
---
 tests/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index a280acc..b4335c5 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -127,7 +127,7 @@ test_fuzz: test_fuzz_certauth test_fuzz_mux test_fuzz_ssl test_fuzz_telnet \
 
 oomtest_SOURCES = oomtest.c
 
-oomtest_LDADD = $(top_builddir)/lib/libgensio.la $(OPENSSL_LIBS) -lpthread
+oomtest_LDADD = $(top_builddir)/lib/libgensio.la $(OPENSSL_LIBS)
 
 noinst_PROGRAMS = oomtest
 
